Ginger/GingerCoreCommon/GeneralLib/General.cs

```diff
@@ -224,5 +224,15 @@ public static void ClearDirectoryContent(string DirPath)
             }
         }
 
+        public static string RemoveInvalidFileNameChars(string Item)
```

Comment:
Give some valid variable name instead of item. Also use pascal casing for method parameter names